app/models/user.rb

```diff
@@ -18,7 +18,7 @@ class User < Common::RedisStore
   attribute :middle_name
   attribute :last_name
   attribute :gender
-  attribute :birth_date, Common::UTCTime
+  attribute :birth_date
```

Comment:
What if you split this into two attributes, one that was a string and another that parsed the string with `Date.parse()`? That way, the data would be stored correctly, and downstream users wouldn't have to know about the right way of parsing the string.